components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -21,6 +21,7 @@
     <string name="error_timeout_message">@string/error_connectionfailure_message</string>
```

Comment:
it's not covered here, but I have a question about the first string [defined at the top of this file](https://github.com/mozilla-mobile/android-components/blob/b308888/components/browser/errorpages/src/main/res/values/strings.xml#L6):

```xml
    <!-- The page html title (i.e. the <title> tag content) -->
    <string name="error_connectionfailure_title">Unable to connect</string>
    <string name="error_connectionfailure_message">
    <![CDATA[
      <ul>
        <li>The site could be temporarily unavailable or too busy. Try again in a few moments.</li>
        <li>If you are unable to load any pages, check your mobile device’s data or Wi-Fi connection.</li>
      </ul>
    ]]>
    </string>
```

Reply:
That's a valid point - I'm not sure there's any value for the localizer in knowing that this string is a page title as well. I'd argue that removing it would be better.

Reply:
Looking at cvan's comment below, instead I'm going to change this to the text below for each title:
```
<!-- The document title for an error page (i.e., the contents of the `<title>` tag). -->
```